lib/handler.js

```diff
@@ -0,0 +1,56 @@
+import Sentry from '@sentry/node'
+
+/**
+ *
+ * @param {object} args
+ * @param {import('pg').Pool} args.pgPool
+ * @param {import('./typings').Logger} args.logger
+ * @returns
+ */
+export const createHandler = ({
+  pgPool,
+  logger
+}) => {
+  return (req, res) => {
+    const start = new Date()
+    logger.request(`${req.method} ${req.url} ...`)
+    handler(req, res, pgPool)
+      .catch(err => errorHandler(res, err, logger))
+      .then(() => {
+        logger.request(`${req.method} ${req.url} ${res.statusCode} (${new Date() - start}ms)`)
+      })
+  }
+}
```

Comment:
nit:


```js
export const createHandler = ({
  pgPool,
  logger
}) => (req, res) => {
  // ...
}
```

Reply:
I find the proposed syntax too terse. It took me a bit to decipher what is `createHandler` returning.

The current version is based on what we have in spark-api:
https://github.com/filecoin-station/spark-api/blob/f905ea21c46fedf2968fa3f797b8fa2169c54d11/index.js#L272-L287